routers/repo/branch.go

```diff
@@ -259,12 +259,13 @@ func loadBranches(ctx *context.Context, page, pageSize int) ([]*Branch, int) {
 		branches = append(branches, deletedBranches...)
 	}
 
-	return branches, len(rawBranches) - 1
+	return branches, totalNumOfBranches - 1
 }
 
 func loadOneBranch(ctx *context.Context, rawBranch *git.Branch, protectedBranches []*models.ProtectedBranch,
 	repoIDToRepo map[int64]*models.Repository,
 	repoIDToGitRepo map[int64]*git.Repository) *Branch {
+	log.Info("loadOneBranch: '%s'", rawBranch.Name)
```

Comment:
At most debug, trace or not all.

```suggestion
	log.Debug("loadOneBranch: '%s'", rawBranch.Name)
```